pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css

```diff
@@ -470,6 +470,10 @@ div#toppanelinner {
   padding: 0;
 }
 
+div#promptPanel > div.prompt-panel {
+    padding: 4px 0 0 8px;

```

Comment:
I think if you add this new rule you can remove `div#promptPanel div.prompt-panel,` from line 468.
And one more think, can you recheck that it looks correctly in scheduler dialog because it also uses prompt panel....and probably PIR.